src/main/kotlin/com/terraformation/backend/accelerator/model/CohortModel.kt

```diff
@@ -0,0 +1,49 @@
+package com.terraformation.backend.accelerator.model
+
+import com.terraformation.backend.db.default_schema.CohortId
+import com.terraformation.backend.db.default_schema.CohortPhase
+import com.terraformation.backend.db.default_schema.ParticipantId
+import com.terraformation.backend.db.default_schema.tables.pojos.CohortsRow
+import com.terraformation.backend.db.default_schema.tables.references.COHORTS
+import org.jooq.Field
+import org.jooq.Record
+
+data class CohortModel<ID : CohortId?>(
+    val id: ID,
+    val name: String,
+    val participantIds: List<ParticipantId>,
+    val phaseId: CohortPhase?
```

Comment:
The naming convention I've been using in model classes is that enums don't have the `Id` suffix on their names. The suffix is there on the `Row` classes just because they're generated from the column names of the underlying tables, but in the context of the model, the phase is a value of type `CohortPhase`, not an ID.

Also, should this be `CohortPhase` instead of `CohortPhase?`?

Reply:
Yes, it needs to be updated now that phase is required in the database.